README.md

```diff
@@ -132,8 +145,10 @@ the following environment variables:
 | `KUBE_PS1_SEPARATOR` | &#124; | Separator between symbol and cluster name |
 | `KUBE_PS1_DIVIDER` | `:` | Separator between cluster and namespace |
 | `KUBE_PS1_SUFFIX` | `)` | Prompt closing character |
-| `KUBE_PS1_CLUSTER_FUNCTION` | No default, must be user supplied | Function to customize how cluster is displayed |
-| `KUBE_PS1_NAMESPACE_FUNCTION` | No default, must be user supplied | Function to customize how namespace is displayed |
+| `KUBE_PS1_CLUSTER_FUNCTION` | (none)\* | Function to customize how cluster is displayed |
+| `KUBE_PS1_NAMESPACE_FUNCTION` | (none)\* | Function to customize how namespace is displayed |
+
+\* _must be user supplied_
```

Comment:
_note_: Not directly related to this PR but I found the middle column unnecessarily large. Happy to revert if you like the full test better.